packages/expo-dev-menu/android/src/debug/java/expo/modules/devmenu/DevMenuHost.kt

```diff
@@ -48,7 +48,7 @@ class DevMenuHost(application: Application) : ReactNativeHost(application) {
 
   override fun getBundleAssetName() = "EXDevMenuApp.android.js"
 
-  override fun getJSMainModuleName() = "index"
+  override fun getJSMainModuleName() = ".expo/.virtual-metro-entry"
```

Comment:
This file will be served by the bare metro from RN. So I think we don't want to change that.